src/shared/wrapWdkDependencies.ts

```diff
@@ -16,13 +16,24 @@ export function areCompatibleWdkDependencies(
 }
 
 export function wrapWdkDependencies(
-  studyAccessApiUrl: string,
-  wdkDependencies: WdkDependencies
+  wdkDependencies: WdkDependencies,
+  studyAccessApiUrl?: string
```

Comment:
Nitpick:

```suggestion
  studyAccessApiUrl: string | undefined
  wdkDependencies: WdkDependencies,
```

Ordering the parameters this way makes it easier to use "partial application" to configure the WDK dependencies.

Reply:
So for legacy mbio, we'd configure `wrapWdkDependencies` by passing `partial(undefined)`.

Reply:
Thanks @jtlong3rd ! Ahh yes I see now the ordering works!

Out of curiosity, is the above a general best practices rule, so whenever i want to add `argument?: type` I should instead add `argument: type | undefined`? Or is it just best for this situation?